pkg/operator/deploy/staticresources/worker/machinehealthcheck.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: machine.openshift.io/v1beta1
```

Comment:
Operator needs to controll this

Reply:
I'll look into the requirements, thanks for the suggestion.